.circleci/config.yml

```diff
@@ -0,0 +1,89 @@
+---
+# all-purpose workspace persister
+persist_to_workspace: &persist_to_workspace
```

Comment:
can use commands with teh 2.1 config to achieve this. 

example here: https://github.com/pantheon-systems/gke-terraform/blob/master/.circleci/config.yml#L23

Personally I like the commands vs the yaml macro.